golem_messages/message/base.py

```diff
@@ -400,11 +401,18 @@ def deserialize_header(cls, data):
         return header
 
     @classmethod
-    def deserialize(cls, msg, decrypt_func, check_time=True, verify_func=None):
+    def deserialize(cls, msg,
+                    decrypt_func,
+                    check_time=True,
+                    verify_sender: bytes = None):

```

Comment:
Maybe `sender_public_key`? I feel it's more explicit than `verify_sender` which suggests a boolean.